lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.tsx

```diff
@@ -64,7 +64,7 @@ export default function OAuth2RedirectErrorApp({
             A Canvas admin needs to edit {"Hypothesis's"} developer key and add
             these scopes:
           </p>
-          <ol className="px-4 list-decimal">
+          <ol className="px-8 list-decimal">
```

Comment:
Perhaps I would add padding to the left only, as the container has its own "global" padding.

```suggestion
          <ol className="pl-8 list-decimal">
```

Reply:
Yes. In this context left-padding makes sense. Note that for lists with translatable content, `padding-inline-start` (`ps-*` in Tailwind) is preferred as it works for both LTR and RTL languages.

Reply:
Yes! Padding start is better. I overlooked tailwind supported this.